packages/load/src/load-typedefs/collect-sources.ts

```diff
@@ -190,18 +183,89 @@ function includeIgnored<
   }
 >({ options, globs }: { options: T; globs: string[] }) {
   if (options.ignore) {
-    const ignoreList = asArray(options.ignore)
-      .map(g => `!(${g})`)
-      .map<string>(unixify);
+    const ignoreList = asArray(options.ignore).map(g => {
+      // FIXME: this seems like the wrong place to do this
+      // should we have the loaders inspect and handle the ignores option?
+      // should we build a list of ignores here and pass that as an argument to resolveGlobs instead of "options"?
+      if (g.startsWith('git:')) {
+        const [, ref, path] = g.split(':');
+        return `git:${ref}:!(${path})`;
+      }
+      return `!(${g})`;
+    });
```

Comment:
perhaps creating an `ignoreGlobs :string[]` variable to pass to `loader.resolveGlobs(globs, ignoreGlobs)` would be less specific here? 🤔 
I think if so, we'd also need to filter the ignore globs by loader support, akin to the grouping for the other globs (via `Map<string:string[]>`)

Reply:
moved to loaders in 092c1b8d4536bb94849a01280f9b88d3310d3338